tests/transformations/subtests/dead-code-dynamic/webpack/index.md

```diff
@@ -1,4 +1,16 @@
 ---
-result: fail
+result: partial
 issue: https://github.com/webpack/webpack.js.org/issues/2684
 ---
+
+Webpack doesn't understand the special destructuring syntax to elimitate dead code:
+
+```js
+const { logCaps } = await import('./utils.js');
+```
+
+But it allows to manually list the exports that are used via magic comment:
+
+```js
+const { logCaps } = await import(/* webpackExports: "logCaps" */ './utils.js');
+```
```

Comment:
@sokra nice! Btw, I struggled to find docs on this. I tried:

- https://webpack.js.org/guides/tree-shaking/
- https://webpack.js.org/guides/ecma-script-modules/
- https://webpack.js.org/guides/code-splitting/

Although I found https://webpack.js.org/api/module-methods/ by googling "webpackExports".

Reply:
https://github.com/webpack/webpack.js.org/issues/4492

Reply:
Cheers! That doesn't stop it being a pass btw, since it _is_ documented.